packages/react/src/dev.tsx

```diff
@@ -157,8 +173,14 @@ ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
     >
       <div className="layout">
         <div>
-          <h2>Switch to default org</h2>
-          <Config />
+          <div>
```

Comment:
Is this a test page?

Reply:
Correct, this is our development page where we render all the different app configs. Not exposed to customers in any way 😄